composer.json

```diff
@@ -17,11 +17,10 @@
         "drupal/better_exposed_filters": "^6.0",
         "drupal/bricks": "^2.1",
         "drupal/cer": "^5.0@beta",
-        "drupal/content_fixtures": "^3.1",
-        "drupal/core": "^10.3",
-        "drupal/core-composer-scaffold": "^10.3",
-        "drupal/core-project-message": "^10.3",
-        "drupal/core-recommended": "^10.3",
+        "drupal/content_fixtures": "^3.2",
```

Comment:
This is not directly related to this pull request, but `drupal/content_fixtures` should be moved to `require-dev`.